src/packet.c

```
@@ -3,295 +3,307 @@
 
     return 0;
 }
 
-/**
- * Encrypt a packet's payload
- */
-int packet_encrypt(packet_t *packet, const uint8_t *key)
-{
-    if (!packet || !key)
-    {
+int packet_encrypt(packet_t* packet, const uint8_t* key) {
```

Cppcheck (2.21.0)

[style] The function 'packet_encrypt' is never used.

(unusedFunction)